src/components/Expression/AutosizeInput/index.tsx

```diff
@@ -39,31 +38,42 @@ InputContainer.displayName = "InputContainer";
 
 export const AutosizeInput = ({
   value,
+  placeholder,
   defaultWidth = 71,
   onChange,
   onCalculationChange,
   type = "text",
   ...props
 }: AutosizeInputProps) => {
-  const inputRef = useRef();
-  const { measureText } = useTextMetrics(inputRef);
-  const currentTextWidth = measureText(value || "");
-  const width =
-    currentTextWidth > defaultWidth ? currentTextWidth : defaultWidth;
+  const inputRef = useRef<HTMLInputElement>();
+  const { measureText, measureTextWithPaddings } = useTextMetrics(inputRef);
+
+  useEffect(() => {
+    if (!inputRef.current) return;
```

Comment:
**nitpick:** Personally when i see return in useEffect it means cleanup function, so I would rather see whole logic wrapped in if statement, but it works fine and can be used like this so it's up to you